src/adapter/mod.rs

```diff
@@ -254,7 +261,7 @@ impl<'a> Adapter<'a> for RustdocAdapter<'a> {
                             actual_type_name,
                             "PlainVariant" | "TupleVariant" | "StructVariant"
                         ),
-                        "ImplOwner" => matches!(actual_type_name, "Struct" | "Enum"),
+                        "ImplOwner" => matches!(actual_type_name, "Struct" | "Enum" | "Union"),
```

Comment:
(GitHub doesn't allow commenting on lines not modified in the PR)

Make sure to also include `ItemEnum::Union` in `supported_item_kind` on line 280 in this file. Otherwise you'll get the exact "unions looked up by name don't exist" bug I mentioned.

Reply:
Shoot, I was grepping for "Enum" so I missed this. Fixed now.

Reply:
It happens! You would have caught it in a test though :) Also, this is why we do code review!